packages/ui/src/assets/svg/arrow-bottom.ts

```diff
@@ -1,10 +1,10 @@
 import { svg } from 'lit'
 
-export const arrowBottomSvg = svg`<svg fill="none" viewBox="0 0 10 10">
+export const arrowBottomSvg = svg`<svg fill="none" viewBox="0 0 14 15">
```

Comment:
Should this be 14x14 ? I noticed that all icons of this size have a box of `14 15`, perhaps there is issue in figma?